app/src/main/java/com/example/sharingang/items/InMemoryItemRepository.kt

```diff
@@ -27,10 +27,13 @@ class InMemoryItemRepository @Inject constructor() : ItemRepository {
         return itemsMap.values.toList()
     }
 
-    override fun getAllItemsLiveData(): LiveData<List<Item>> {
+    override fun items(): LiveData<List<Item>> {
         return itemsLiveData
     }
 
+    override suspend fun refreshItems() {
```

Comment:
What does this function do, is it intended to be empty?

Reply:
Yeah, the idea is that since this is just mocking the behavior, there's no need to actually refresh anything.